extension/user/checkAuthSuccess.js

```diff
@@ -0,0 +1,17 @@
+const {EAUTH, customError} = require('../error')
+/**
+ * @typedef {Object} CheckAuthSuccessArgs
+ * @property {boolean} authSuccess
+ */
+
+/**
+ * @param {SDKContext} context
+ * @param {CheckAuthSuccessArgs} input
+ * @param {function} cb
+ */
+module.exports = function (context, input, cb) {
+  if (input.authSuccess !== true) {
+    return cb(customError(EAUTH, 'Auth failed'))
```

Comment:
This should be an internal error, because as an external user oder theme developer, you can't do anything about it.
Some logging of this error should be introduced.